src/SQLite.Net/SQLiteCommand.cs

```diff
@@ -473,7 +473,7 @@ private object ReadCol(IDbStatement stmt, int index, ColType type, Type clrType)
                 {
                     return new DateTime(_sqlitePlatform.SQLiteApi.ColumnInt64(stmt, index), DateTimeKind.Utc);
                 }
-                return DateTime.Parse(_sqlitePlatform.SQLiteApi.ColumnText16(stmt, index), CultureInfo.InvariantCulture);
+                return DateTime.Parse(_sqlitePlatform.SQLiteApi.ColumnText16(stmt, index), CultureInfo.InvariantCulture).ToUniversalTime();
```

Comment:
better to use `return DateTime.Parse(_sqlitePlatform.SQLiteApi.ColumnText16(stmt, index), CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal)`